laze-project.yml

```
@@ -1318,6 +1318,9 @@ builders:
   - name: espressif-esp32-devkitc
     parent: esp-wroom-32
 
+  - name: espressif-esp32-c3-lcdkit
+    parent: esp32c3
```

This needs an extra laze context to represent the ESP32 hardware module (similar to `esp32-c6-wroom-1`):
```suggestion
    parent: esp32-c3-mini-1
```